examples/api/python/utils.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+###############################################################################
+# Top contributors (to current version):
+#   Yoni Zohar
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# Utility Methods, translated from examples/api/utils.h
+##
+
+import pycvc5
+from pycvc5 import kinds
+
+# Get the string version of define-fun command.
+# @param f the function to print
+# @param params the function parameters
```

Comment:
Missing a param for `sort`. Speaking of which, can't you get the return sort from the function `f`?

Reply:
> Missing a param for sort
Thanks! fixed here: 960fbb2

> can't you get the return sort from the function f?
I am not sure, I translated this from the cpp example. @ajreynol do you have any preference here? Either way I recommend keeping the cpp and python similar in this case.

Reply:
I think its fine to remove the sort from both. Indeed it is redundant.

Reply:
done here: 8f1d614